app/Filament/Widgets/ServiceRequestWidget.php

```diff
@@ -80,7 +80,12 @@ protected function getStats(): array
 
     private function calculateOpenServiceRequestStats(Carbon $intervalStart): array
     {
-        $openStatusIds = ServiceRequestStatus::tap(new ClassifiedAs(SystemServiceRequestClassification::Open))->pluck('id');
+        $openStatusIds = ServiceRequestStatus::tap(new ClassifiedIn([
+            SystemServiceRequestClassification::Open,
+            SystemServiceRequestClassification::InProgress,
+            SystemServiceRequestClassification::Waiting,
+            SystemServiceRequestClassification::Custom,
```

Comment:
What do you think about returning this list from a static method of `SystemServiceRequestClassification` like `getUnclosedClassifications`.

That way if we ever add a new classification that would technically be a part of this list, we don't have to track down everywhere we put it.

Reply:
Yep that's been added !

Thank you.